src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry/Commands/GetAzureContainerRegistryWebhookEvent.cs

```diff
@@ -0,0 +1,73 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Management.ContainerRegistry.Models;
+
+namespace Microsoft.Azure.Commands.ContainerRegistry
+{
+    [Cmdlet(VerbsCommon.Get, ContainerRegistryWebhookEventNoun)]
+    [OutputType(typeof(IList<PSContainerRegistryWebhookEvent>))]
+    public class GetAzureContainerRegistryWebhookEvent : ContainerRegistryCmdletBase
+    {
+        [Parameter(Position = 0, Mandatory = true, ParameterSetName = ListWebhookEventsByNameResourceGroupParameterSet, HelpMessage = "Webhook Name.")]
+        [ValidateNotNullOrEmpty]
+        [Alias(WebhookNameAlias)]
+        public string Name { get; set; }
```

Comment:
Good catch.

Reply:
Do you need to rename `Name` to `ReplicationName` in replication commands?

Reply:
No. Since the command name is webhookevent, Name may be confusing. It's not clear if it means webhook name or webhook event name. We don't have this problem in Replication commands.